packages/web/server/index.ts

```diff
@@ -154,13 +155,21 @@ function wwwRedirect(req, res, nextAction) {
   })
 
   server.get('/proxy/medium', async (_, res) => {
-    const articlesdata = await getFormattedMediumArticles()
-    res.json(articlesdata)
+    try {
+      const articlesdata = await getFormattedMediumArticles()
+      res.json(articlesdata)
+    } catch (e) {
+      res.status(e.statusCode || 500).json({ message: e.message || 'unknownError' })
+    }
   })
 
   server.get('/proxy/events/', async (_, res) => {
-    const events = await getFormattedEvents()
-    res.json(events)
+    try {
+      const events = await Promise.race([getFormattedEvents(), abort('Events from Airtable')])
```

Comment:
Just for consistency, could you do the abort in the same place for both of these proxy routes? (either in the getFormatted function or at the top level here?